db/migrate/20150512083842_remove_old_translation_from_listing_unit.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveOldTranslationFromListingUnit < ActiveRecord::Migration
+  def up
+    remove_column :listing_units, :translation_key
+  end
+
+  def down
+    add_column :listing_units, :translation_key, :string, limit: 64, after: :quantity_selector
+  end
+end
```

Comment:
Because of the 2 min preboot delay, there's a change that removing a column may break the system for 2 minutes. The reason is that when you run the migration we remove the column, but the old code which still thinks that there is the `translation_key` may break.

Here's a good blog post about the different things you have to consider when doing zero downtime deployments in Rails. http://blog.codeship.com/rails-migrations-zero-downtime/ See the chapter "Removing columns".

I'd suggest that you add the `self.columns` method to the `ListingUnit` model as described in the article.